app/Kernel.php

```diff
@@ -32,6 +36,8 @@
      */
     public function __construct($environment, $debug)
     {
+        $this->request = Request::createFromGlobals();
```

Comment:
Isn't the kernel used in CLI commands? I'd like to see the import locale cli command tested before merging this